app/grandchallenge/evaluation/templates/evaluation/evaluation_list.html

```diff
@@ -71,12 +71,7 @@ <h2>Submissions and Evaluations for {{ phase.title }}</h2>
                         </td>
                     {% endif %}
                     <td>
-                    <span class="badge badge-{{ evaluation.status_context }}">
-                        {% if evaluation.animate %}
-                            <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
-                        {% endif %}
-                        {{ evaluation.get_status_display }}
-                    </span>
+                        {% include "algorithms/job_status_badge_detail.html" with object=evaluation %}
```

Comment:
Note this is for the evaluation status and not the job status, you have not implemented this yet and should follow with another PR.

Reply:
Yes, correct. I was looking to see if it worked because this also inherits from `ComponentJob`. (I was a little surprised that it actually works, but maybe I did not test it thoroughly enough.)

Will revert this.